### PR TITLE
dotnet-svcutil test fix

### PR DESCRIPTION
**Pull request by imcarolwang**

### CODE REVIEW

File: src/dotnet-svcutil/lib/tests/Baselines/OptionsRoundtrip/SvcutilOptions/dotnet-svcutil.svcutil.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "providerId": "Microsoft.Tools.ServiceModel.Svcutil",
-  "version": "99.99.99.1313",
+  "version": "99.99.99",
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/tests/Baselines/OptionsRoundtrip/SvcutilOptions/dotnet-svcutil.svcutil.json:3]: Is this change necessary?

File: src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/nuget.config [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key = "svcutilTestFeed" value="$repositoryRoot$/artifacts/Packages/Release/Shipping" />
+    <add key = "svcutilTestFeed" value="$repositoryRoot$/artifacts/packages/Release/Shipping" />
     <add key = "nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/nuget.config:6]: is this change necessary?

File: src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/SvcutilBootstrapper/SvcutilBootstrapper.csproj [hunk 3]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="dotnet-svcutil-lib">
-      <HintPath>$resultPath$/TestResults/TFMBootstrap/tfm100/bin/Debug/netcoreapp3.1/dotnet-svcutil-lib.dll</HintPath>
+      <HintPath>$resultPath$/TestResults/TFMBootstrap/tfm100/bin/Debug/net5.0/dotnet-svcutil-lib.dll</HintPath>
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/SvcutilBootstrapper/SvcutilBootstrapper.csproj]: With this change, will there be any issue for this to run on 3.1?  Will there be any backward compatibility issue?

File: src/dotnet-svcutil/lib/tests/src/E2ETests.cs [hunk 4]
```diff
@@ -83,7 +83,8 @@ public void TFMBootstrap(string testCaseName, string targetFramework)
         {
             this_TestCaseName = "TFMBootstrap";
             TestFixture();
-            InitializeE2E(testCaseName, createUniqueProject: true, targetFramework: "netcoreapp3.1", sdkVersion: "3.1.101");
+            //InitializeE2E(testCaseName, createUniqueProject: true, targetFramework: "netcoreapp3.1", sdkVersion: "3.1.101");
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/tests/src/E2ETests.cs]: do we need to leave this in, or should this be deleted?

File: src/dotnet-svcutil/lib/tests/src/TestInit.cs [hunk 5]
```diff
@@ -117,10 +129,16 @@ public ClassFixture()
 
             var vsTestsRoot = GetSTestRootDir();
 
-            g_SdkVersion = "3.1.100";
+            g_SdkVersion = "5.0.100";
```
Comment (HongGit) [inline src/dotnet-svcutil/lib/tests/src/TestInit.cs:132]: what happens if we need to run test on version 3.1?

Comment (imcarolwang) [reply src/dotnet-svcutil/lib/tests/src/TestInit.cs:132]: When we need to test version 3.1, the 3.1 version is passed via global.json, and it's get at runtime from test code by executing command "dotnet --version", I've validated manually the test passed.

File: src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/nuget.config (hunk 2 above)
Comment (imcarolwang) [reply src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/nuget.config:6]: Yes, this is to align with the actual result generated on Linux OS which path name is case-sensitive.

File: src/dotnet-svcutil/lib/tests/Baselines/OptionsRoundtrip/SvcutilOptions/dotnet-svcutil.svcutil.json (hunk 1 above)
Comment (imcarolwang) [reply src/dotnet-svcutil/lib/tests/Baselines/OptionsRoundtrip/SvcutilOptions/dotnet-svcutil.svcutil.json:3]: The original baseline saved only replaced partial version number of a result output, with the change applied, we get clear and common match in baseline comparison for the generated result.

File: src/dotnet-svcutil/lib/src/Shared/FrameworkInfo.cs [hunk 6]
```diff
@@ -36,11 +36,11 @@ public static FrameworkInfo Parse(string fullFrameworkName)
             }
 
             // framework spec form: 'netcore1.5' or 'net452'
-
+            // framework spec form: 'net5.0'
             for (int i = 0; i < fullFrameworkName.Length; i++)
             {
                 char c = fullFrameworkName[i];
-                if (Char.IsNumber(c))
+                if (char.IsNumber(c))
```
[the remark this answers is not in the document]
Comment (imcarolwang) [reply src/dotnet-svcutil/lib/src/Shared/FrameworkInfo.cs:43]: It's said that "char" is alias of "System.Char", so the change here is not necessary.

File: eng/Versioning.props [hunk 7]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <GenerateAssemblyInfo Condition="'$(GenerateAssemblyInfo)'==''">true</GenerateAssemblyInfo>
     <Ship_SvcUtilXmlSerPackages Condition="'$(Ship_SvcUtilXmlSerPackages)'==''">false</Ship_SvcUtilXmlSerPackages>
-    <Ship_SvcUtilPackages Condition="'$(Ship_SvcUtilPackages)'==''">false</Ship_SvcUtilPackages>
+    <Ship_SvcUtilPackages Condition="'$(Ship_SvcUtilPackages)'==''">true</Ship_SvcUtilPackages>
```
[the remark this answers is not in the document]
Comment (imcarolwang) [reply eng/Versioning.props:5]: Ok. I was adding that to build the tool for test.

File: src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/SvcutilBootstrapper/SvcutilBootstrapper.csproj (hunk 3 above)
Comment (imcarolwang) [reply src/dotnet-svcutil/lib/tests/Baselines/TFMBootstrap/tfm100/SvcutilBootstrapper/SvcutilBootstrapper.csproj]: The baseline is fixed in a followed commit.

File: src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs [hunk 8]
```diff
@@ -45,6 +45,12 @@ internal class TargetFrameworkHelper
                 ProjectDependency.FromPackage("System.ServiceModel.Http", "4.4.*"    ),
                 ProjectDependency.FromPackage("System.ServiceModel.NetTcp", "4.4.*"  ),
                 ProjectDependency.FromPackage("System.ServiceModel.Security", "4.4.*"),
+            } },
+            {new Version("5.0"), new List<ProjectDependency> {
+                ProjectDependency.FromPackage("System.ServiceModel.Duplex", "4.7.*"  ),
+                ProjectDependency.FromPackage("System.ServiceModel.Http", "4.7.*"    ),
+                ProjectDependency.FromPackage("System.ServiceModel.NetTcp", "4.7.*"  ),
+                ProjectDependency.FromPackage("System.ServiceModel.Security", "4.7.*"),
             } }
```
[the remark this answers is not in the document]
Comment (imcarolwang) [reply src/dotnet-svcutil/lib/src/Shared/TargetFrameworkHelper.cs:54]: This is currently keeping unchanged.

File: eng/Versioning.props (hunk 7 above)
[the remark this answers is not in the document]
Comment (imcarolwang) [reply eng/Versioning.props]: Updated to the original value (false).

**Comment by HongGit** [comment]
@imcarolwang could you please rebase this PR?

**Review by HongGit — APPROVED** [review]
